### PR TITLE
New In Chrome 57 Updates post

### PR DESCRIPTION
**Pull request by petele**
Before merging:
* [ ] Update video link

### CODE REVIEW

File: src/content/en/updates/2017/03/nic57.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+<div class="clearfix"></div>
+
+## CSS Grid Layout {: #gridlayout }
+
+<img src="/web/updates/images/2017/03/nic57/grid-layout.png" class="attempt-right">
+
+Flexbox is a powerful layout tool, it makes many complex layouts possible,
+but it can only do layout in one dimension. Chrome 57 adds support for
+[`display: grid`](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids)
+- the new [CSS Grid Layout specification](https://www.w3.org/TR/css3-grid-layout/),
```
Comment (jpmedley) [inline src/content/en/updates/2017/03/nic57.md]: The phrase 'the new CSS Grid Layout specification' should be punctuated the same on both ends. Either use commas at both ends or dashes at both ends. If you use dashes, they should be m-dashes. (The entity is &amp;mdash&semi;).

File: src/content/en/updates/2017/03/nic57.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+adding a powerful new tool for creating two-dimensional grid-based layout
```
Comment (jpmedley) [inline src/content/en/updates/2017/03/nic57.md]: The word 'grid-based' is inconsistently hyphenated. Please check throughout. I don't care whether you keep or lose the hyphen, but keeping it is usually the better option in a technical document.

File: src/content/en/updates/2017/03/nic57.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+book_path: /web/updates/_book.yaml
+description: What's new in Chrome 57 for developers? Grid based layouts, Media Session API and more.
+
+{# wf_published_on: 2017-03-08 #}
+{# wf_updated_on: 2017-03-08 #}
+{# wf_featured_image: /web/updates/images/generic/new-in-chrome.png #}
+{# wf_tags: chrome57,new-in-chrome,css,progressive-web-apps,media,grids #}
+{# wf_featured_snippet: With Chrome 57, you can now use <code>display: grid</code> for grid based layouts, use the <b>media session API</b> to customize the lock screen and notifications with information about the media being played and more. Pete LePage has all the details and how you can use these new developer features in Chrome 57! #}
+
+# New In Chrome 57 {: .page-title }
```
Comment (jpmedley) [inline src/content/en/updates/2017/03/nic57.md]: 'In' should be lowercase. (I want to clarify that I'm not mistaking this for a level 2 or lower. The rules for title case specify that prepositions be lowercase.)

File: src/content/en/updates/2017/03/nic57.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+  notifications using the new [Media Session API](#mediasession).
+* And there are plenty [more](#andmore)!
+
+<div class="clearfix"></div>
+
+I’m Pete LePage, let’s dive in and see what’s new for developers in Chrome 57! 
+
+<div class="clearfix"></div>
+
+## CSS Grid Layout {: #gridlayout }
```
Comment (jpmedley) [inline src/content/en/updates/2017/03/nic57.md]: 'Grid layout' should be lower case.

File: src/content/en/updates/2017/03/nic57.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  properties.
+* The [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)
+  [Response class](https://developer.mozilla.org/en-US/docs/Web/API/Response)
+  now supports the
+  [`.redirected`](https://developer.mozilla.org/en-US/docs/Web/API/Response/redirected)
+  attribute to help avoid untrustworthy responses and reduce the risk of open
+  redirectors. 
+* All `-webkit-` prefixed IndexedDB global aliases have been removed, after
+  their deprecation in M38. 
+* And one of my favorites -- new
```
Comment (jpmedley) [inline src/content/en/updates/2017/03/nic57.md]: Use an m-dash.

File: src/content/en/updates/2017/03/nic57.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+were instrumental on landing the new CSS Grid and `caret-color` features.
+
+## Stay up to date
+
+
+If you want to stay up to date with Chrome and know what’s coming, be sure to
+[subscribe](https://goo.gl/6FP1a5), follow 
+[@ChromiumDev](//twitter.com/chromiumdev) on Twitter and be sure to check out the
+[videos from the Chrome Dev Summit](https://www.youtube.com/playlist?list=PLNYkxOF6rcIBTs2KPy1E6tIYaWoFcG3uj)
+for a deeper dive into some of the awesome things the Chrome team is working on.
```
Comment (jpmedley) [inline src/content/en/updates/2017/03/nic57.md]: I realize this is essentially a transcript of a video, but this paragraph has issues. Here's what I'd have done automatically if not for that:
* Line 119: add 'and' before 'follow'.
* LIne 120: end sentence after 'Twitter' and start new sentence with 'and'. (Yes, that's allowed, as long as you know what you're doing.)

Comment (petele) [inline src/content/en/updates/2017/03/nic57.md]: @jpmedley - tweaked this section, please take a look at the updated version.

**Pull request merged**